### PR TITLE
New tutorials

### PR DESCRIPTION
**Pull request by datascienceclubUVU**

### CODE REVIEW

File: tutorials/SQL/Views.html [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+                        <h2 class="mt-5 px-4 px-sm-3 border-start border-3 border-secondary">How to Create Views</h2>
+                        <p class="pt-3">Now that you have a clear understanding of how a View differs from a table and when to use views, let's take
+                            a look at how to create Views. Watch the following video to get a better understanding of this concept:
+                        </p>
+                        <ul>
+                            <a href="https://www.youtube.com/watch?v=vLLkNI-vkV8&t=65s">SQL Views in 4 Minutes</a>
+                        </ul>
+                        <h4 class="pt-4">Summary:</h4>
+                        <ul>
+                            <li class="pt-1">Compared to other DDL commands, the <code>CREATE VIEW </code>command is pretty straight forward.
```
Comment (jbeBan) [inline tutorials/SQL/Views.html:158]: The word `straight forward` on line `158` should be `straightforward`.

File: tutorials/SQL/Views.html [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+                        </p>
+                        <ul>
+                            <a href="https://docs.vmware.com/en/VMware-Tanzu-Greenplum/7/greenplum-database/GUID-admin_guide-ddl-ddl-view-best-practices.html">Best Practices When Creating Views</a>
+                        </ul>
+                        <h4 class="pt-4">Summary:</h4>
+                        <ul>
+                            <li class="pt-1">"If a subquery is associated with a single query, consider using the <code> WITH </code> Clause of the <code>
+                                 SELECT </code> command instead of creating a seldom-used view." (docs.vmware.com, 2023)
+                            </li>
+                            <li class="pt-4">"Defining a denormalized 'world' view. A view that joins a large number of database table that is used
```
Comment (jbeBan) [inline tutorials/SQL/Views.html:179]: The word `table` should be pluralized on line `179`.

**Comment by datascienceclubUVU** [comment]
made requested changes and pushed to remote repo

**Review by jbeBan — APPROVED** [review]
Looks good now!

**Pull request merged**